content/en/service_management/events/correlation/patterns.md

```diff
@@ -12,12 +12,25 @@ To get you started, Datadog automatically suggests [pattern-based correlations][
 {{< img src="service_management/events/correlation/pattern/recommended_patterns_preview.png" alt="Correlation recommended patterns with the preview panel showing potential cases the pattern would create" style="width:100%;" >}}
 
 
-## Adjusting a pattern
+## Create a pattern
 
-To create or adjust a pattern-based correlation:
+To create a pattern:
 1. Navigate to [Correlation][1].
-1. Click **+ Add a Pattern**, at the top of the Pattern table. This opens a pattern configuration page.
-1. From the dropdown menu, select the alerting sources you want to correlate events for. 
+1. Click **+ Add a Pattern**, at the top of the Pattern table. This opens pattern configuration page with out-of-the-box suggested patterns on the left side , and pattern output preview on the right side. 
+1. You can adjust the suggested pattern by clicking **+ Continue With Pattern**(this will take you to the pre-populated configuration page for additonal tuning) or choose to create your own by clicking **+ Personalize From Scratch**
+
+Events first get deduplicated to alerts based on event aggregation key, alerts get correlated to case based on configuration 
+{{< img src="service_management/events/correlation/correlation_helper.mp4" alt="Dedupe and correlate events" video=true >}}
```

Comment:
Best practice is not to have text information on videos for accessibility. Also, if possible, recommend using deduplicated, instead of de-deplicated.

```suggestion
{{< img src="service_management/events/correlation/correlation_helper.mp4" alt="When a pattern matches defined alerting sources, a case is created. Source alerts are added, and its events are de-duplicated for 1 hour before the process repeats in a new case. You can modify these in advanced settings" video=true >}}
```